wallet-new/src/Cardano/Wallet/Server/Plugins.hs

```diff
@@ -41,6 +46,7 @@ import qualified Cardano.Wallet.Kernel.Diffusion as Kernel
 import qualified Cardano.Wallet.Kernel.Mode as Kernel
 import qualified Cardano.Wallet.Server as Server
 import qualified Cardano.Wallet.WalletLayer.Kernel as WalletLayer.Kernel
+import           Data.Aeson (encode)
```

Comment:
Didn't get how these imports are grouped.

Reply:
Unfortunately, there's no way to enforce rules on imports, that would require the linter to have some semantics notions about them. So, the best we can do is to enforce this -- which also gets complicated in merges and rebases :| 

Please, follow as much as possible @erikd's suggestion on that.